Four-column / "largeDesktop" breakpoint and layout (#529)
commercetools/commercetools-docs-kit: websites/api-docs-smoke-test/src/content/regular-desktop-layout.mdx

---
title: Regular Desktop Layout

> description: Test components that can flow into two content columns when not allowed to

Reviewer: ```suggestion
description: Test components that can not flow into two content columns when the option is disabled
```

wideLayout: false
---

<Subtitle>

> Tests the responsive behavior in "largeDesktop" responsive sizes. It has the `widelayout` frontmatter set to `true`.

Reviewer: ```suggestion
Tests the responsive behavior in "largeDesktop" responsive sizes. It has the `widelayout` frontmatter set to `false`.
```
